wiki/Chat_Console/en.md

```diff
@@ -1,275 +1,106 @@
+| /j [channel] or /join [channel] | Joins the specified channel | /join #lobby | (#lobby tab is opened) | 
+| /p or /part | Leaves the current channel you're parting or leaving. | /part | n/a | 
+| /unignore [user] | Stop ignoring this user for this session. | /unignore Amigo | You may now hear Amigo. (Your chat console will allow any comment made by Amigo available to your chat console) | 
+| /me [action] | Perform a third-person action. | /me is at home | *John is at home | 
+| /msg [user] [msg] | Send a private message to [user]. | /msg Amigo I am sick at home. | (At Amigo tab) John: I am sick at home. | 
+|/np| Print to chat the current song you are listening to or playing.| /np| (If playing) *John is playing [Peter Lambert - osu! tutorial\[Gameplay Basics\]](https://osu.ppy.sh/b/22538) [Game Modes](/wiki/FAQ#Alternative_Game_Modes) \~[Relax/Autopilot](/wiki/Game_Modifiers#Special_Mods)\~ +[Difficulty increasing mod](/wiki/Game_Modifiers#Difficulty_Increasing_Mods) -[Difficulty decreasing mod](/wiki/Game_Modifiers#Difficulty_Reduction_Mods)/[SpunOut](/wiki/Game_Modifiers#Spun_Out) [xK](Game_Modifiers#xK) (If doing nothing, based on jukebox) *John is listening to [Peter Lambert - osu! tutorial](https://osu.ppy.sh/b/22538) (If spectating or Autoplay) *John is watching [Peter Lambert - osu! tutorial\[Gameplay basics\]](https://osu.ppy.sh/b/22538) [Game Modes](/wiki/FAQ#Alternative_Game_Modes) \~[Relax/Autopilot](/wiki/Game_Modifiers#Special_Mods)\~ +[Difficulty increasing mod](/wiki/Game_Modifiers#Difficulty_Increasing_Mods) -[Difficulty decreasing mod](/wiki/Game_Modifiers#Difficulty_Reduction_Mods)/[SpunOut](/wiki/Game_Modifiers#Spun_Out) [xK](Game_Modifiers#xK) [Autoplay](/wiki/Game_Modifiers#Auto)|
+| /reply or /r | Reply to the last received private message. | /r Do you know any good doctor? | (At Amigo tab) [Previous comments] John: I am sick at home. Amigo: Really? John: Do you know any good doctor? | 
+| /savelog | Saves current chat tab to a text file. | /savelog | (A folder called "Chat" will be created at the osu! directory which will contain all the future chat tab saves) | 
+| /watch [user] | Start spectating [user]. | /watch Amigo | *Started spectating Amigo. (When Amigo plays a beatmap that you have, you will spectate his play [after some buffering] with your name on the left of Amigo screen) | 
+| /nopm | Toggle to allow private messages either from everyone or friends only | /nopm | (A pop-up banner will appear at the center detailing you are allowing everyone/friends only for private messages) | 
```

Comment:
Might want to add that doing `/nopm` again will allow private messages from everyone again.